packages/rancher-istio-1.9/package.yaml

```diff
@@ -1,2 +1,2 @@
 url: local
-version: 100.0.0
```

Comment:
If I understand it correctly, we want to keep the rancher-istio-1.9 chart identical to the dev-v2.5 branch, right? 
If so, could we make this file identical to its counterpart in [dev-v2.5](https://github.com/rancher/charts/blob/dev-v2.5/packages/rancher-istio-1.9/package.yaml)

Reply:
I just updated the PR to include why I believe we do not need to do that:

Essentially the version that is in production right now is istio 1.9.500 and kiali 1.32.100. Since then there have been two PRs that made changes to the istio chart but both are due to chart build script changes and not actual istio or kiali upstream changes (8845d9a#diff-8c8f823e4ea6b99f20a772bee097b7fcc47e8893b26b51f7f1a74fb70a54aa94
09f8f4b#diff-8c8f823e4ea6b99f20a772bee097b7fcc47e8893b26b51f7f1a74fb70a54aa94). With this in mind, I think this is just part of the divergence that we will start seeing from the two branches and we should only keep them the same up to what has been released in production.

Reply:
I agree with you. 
And here is my 2 cents:
- we are not changing or editing the released chart in this PR
- either your current changes and my "keeping identical to the dev-2.5" approach will generate the same charts with the same versions
- if we have no plan to release new 1.9.x charts from this branch (dev-v2.6), then I thought it might be easier to just leave the files identical to the dev-v2.5 branch. 

But your changes and the reasons make sense to me as well.

Reply:
That makes sense - I can upgrade it to match the dev-v2.5 branch exactly.

Reply:
@jiaqiluo I just realized - dev 2.5 has packageVersion and 2.6 is using version - wont that cause a problem if I try to make it exactly like 2.5?

Reply:
@brendarearden It should continue working. packageVersion is stilled supported, we just avoid it in new charts.